nimsuggest/tests/toutline_generic.nim

```diff
@@ -0,0 +1,11 @@
+from typetraits import supportsCopyMem
+
+proc tgeneric[T](x: var T) =
```

Comment:
The tests needs to be reduced further. It shouldn't depend on `supportsCopyMem`, which is unrelated to the problem.

Reply:
can't think how to detect it invoke `safeSemExpr` or not.

Reply:
You force the circumstances that previously led to `semIdeForTemplateOrGenericCheck` setting `requiresCheck` to true, and then ensure that a local within the generic is not reported as part of the `outline` command.

Reply:
no idea how to do that, as `semIdeForTemplateOrGeneric` used for `ideSug, ideCon`, how I test it under `outline`

Reply:
`semIdeForTemplateOrGenericCheck` only wrongfully set `requiresCheck` to true when there's either:
* a cursor is placed in the body
* no cursor is provided to the command and there's a node with missing line information in the body

The most simple way to (currently) force a node with missing line information is to use a statement that has an optional slot filled with `nkEmpty` by the parser.

I've made a separate suggestion for how to improve the test.